src/components/inputs/sl-alpha-input.js

```diff
@@ -1,21 +1,8 @@
-'use strict';
-
 import React from 'react';
-import PropTypes from 'prop-types';
-import NumberInput from './number-input';
-
-class SLAlphaInput extends React.PureComponent {
-  static propTypes = {
-    id: PropTypes.string.isRequired,
-    theme: PropTypes.object.isRequired,
-    value: PropTypes.number.isRequired,
-    onChange: PropTypes.func.isRequired,
-    readOnly: PropTypes.bool
-  };
+import { NumberInput } from './number-input';
 
-  render() {
-    return <NumberInput {...this.props} min={0} max={100} />;
-  }
+function SLAlphaInput(props) {
```

Comment:
Is there a reason you removed proptypes? function components can still have 'em.

Reply:
I removed them because this component is really just a passthrough to `NumberInput` which uses proptypes.